src/auth/auth.module.ts

```
@@ -49,12 +52,18 @@ import { SkillVerification } from './entities/skills-verification.entity';
       inject: [ConfigService],
       useFactory: async (configService: ConfigService) => {
         const secret = configService.get<string>('JWT_SECRET');
+        const refreshSecret = configService.get<string>('JWT_REFRESH_SECRET');
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)